src/app/containers/ChartbeatAnalytics/canonical/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { string, bool, number } from 'prop-types';
+import Helmet from 'react-helmet';
+
+const CanonicalChartbeatBeacon = ({
+  domain,
+  type,
+  sections,
+  cookie,
+  chartbeatUID,
+  useCanonical,
+}) => (
+  <Helmet>
+    <script type="text/javascript">
+      {`
+        (function(){
+          var _sf_async_config = window._sf_async_config = (window._sf_async_config || {});
+          _sf_async_config.uid = ${chartbeatUID};
+          _sf_async_config.domain = "${domain}";
+          _sf_async_config.useCanonical = ${useCanonical};
+          _sf_async_config.useCanonicalDomain = ${useCanonical};
+          _sf_async_config.contentType = "${type}";
+          _sf_async_config.sections = "${sections}";
+          _sf_async_config.idSync = {
+           bbc_hid: "${cookie}"
+          };
+          function loadChartbeat() {
+           var e = document.createElement('script');
+           var n = document.getElementsByTagName('script')[0];
+           e.type = 'text/javascript';
+           e.async = true;
+           e.src = '//static.chartbeat.com/js/chartbeat.js';
+           n.parentNode.insertBefore(e, n);
+          }
+          loadChartbeat();
```

Comment:
While the chartbeat script itself if async, this code runs synchronously, blocking paint. Could this be improved? Perhaps making it run async, or stopping this running server side, so it only loads during the bundle hydration?

Reply:
Thanks @dr3, I have added the `async` attribute to the script tag